README.md

```diff
@@ -48,16 +57,18 @@ JSON endpoints for convenience:
 #### Monitoring
 
 Substrate Analytics provides a `/metrics` endpoint for Prometheus.
+TODO: can you elaborate on what people would use this for? Would it be like a "Prometheus aggregator", providing data for all nodes connected to `analytics`? Don't nodes provide their data to Prometheus directly as well?
```

Comment:
This is the metrics we currently collect, including some system metrics via sysinfo crate - not sure if this should be removed.

```
# HELP substrate_message_count Number of binary and text messages received - (does not include PING/PONG messages)
# TYPE substrate_message_count counter
substrate_message_count 1967
# HELP nodes_connected_count Total number of WS connections made since launch.
# TYPE nodes_connected_count counter
nodes_connected_count 4
# HELP nodes_dropped_count Total number of WS connections dropped since launch.
# TYPE nodes_dropped_count counter
nodes_dropped_count 0
# HELP current_substrate_connections Number of WS substrate connections sending data.
# TYPE current_substrate_connections gauge
current_substrate_connections 4
# HELP substrate_bytes_received Total bytes received in binary and text WS messages from substrate clients.
# TYPE substrate_bytes_received counter
substrate_bytes_received 439853
# HELP bytes_received Total bytes received in binary and text WS messages.
# TYPE bytes_received counter
bytes_received 2188421709
# HELP bytes_sent Total bytes received in binary and text WS messages.
# TYPE bytes_sent counter
bytes_sent 2115408029
# HELP requests Number of get requests to non WS routes, also excluding metrics route.
# TYPE requests counter
requests 0
# HELP current_feed_connections Number of WS feed connections consuming data.
# TYPE current_feed_connections gauge
current_feed_connections 0
# HELP feeds_connected Number of WS connections for live feed
# TYPE feeds_connected counter
feeds_connected 0
# HELP feeds_disconnected Number of WS disconnections for live feed
# TYPE feeds_disconnected counter
feeds_disconnected 0
# HELP load_avg_one System load average one minute
# TYPE load_avg_one gauge
load_avg_one 0
# HELP load_avg_five System load average five minutes
# TYPE load_avg_five gauge
load_avg_five 0.13
# HELP load_avg_fifteen System load average fifteen minutes
# TYPE load_avg_fifteen gauge
load_avg_fifteen 0.26
# HELP total_mem Total system RAM (KiB)
# TYPE total_mem gauge
total_mem 65778768
# HELP used_mem Used system RAM (KiB)
# TYPE used_mem gauge
used_mem 3091428
# HELP total_swap Total system swap (KiB)
# TYPE total_swap gauge
total_swap 2097148
# HELP used_swap Used swap (KiB)
# TYPE used_swap gauge
used_swap 0
```